tubular/edx_api.py

```diff
@@ -88,17 +113,19 @@ def _giveup_on_unexpected_exception(exc):
     """
     Giveup method that gives up backoff upon any unexpected exception.
     """
-    return not ( (500 <= exc.response.status_code < 600 and exc.response.status_code != 504) or exc.response.status_code == 104)
+    return not ((
+                        500 <= exc.response.status_code < 600 and exc.response.status_code != 504) or exc.response.status_code == 104)
```

Comment:
please, check the code formatting here, it looks ugly a bit )

Reply:
changed the local reformat code, but I did not see 
thx!